web/src/components/navbar/desktop/NavBarDesktopQuickLinks.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { NavBarDesktopQuickLinks } from "@/components/navbar/desktop/NavBarDesktopQuickLinks";
+import { getMergedConfig } from "@/contexts/configContext";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("<NavBarDesktopQuickLinks />", () => {
+  const Config = getMergedConfig();
+
+  window.open = jest.fn();
+
+  it("renders the quick links as expected", () => {
+    render(<NavBarDesktopQuickLinks />);
+    expect(
+      screen.getByText(Config.navigationDefaults.navigationQuickLinks.navBarPlanText)
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText(Config.navigationDefaults.navigationQuickLinks.navBarStartText)
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText(Config.navigationDefaults.navigationQuickLinks.navBarOperateText)
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText(Config.navigationDefaults.navigationQuickLinks.navBarGrowText)
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText(Config.navigationDefaults.navigationQuickLinks.navBarUpdatesText)
+    ).toBeInTheDocument();
+    expect(screen.getByTestId("navbar-search-icon")).toBeInTheDocument();
+  });
+
+  it("renders the plan quick link and redirects correctly", async () => {
+    render(<NavBarDesktopQuickLinks />);
+    fireEvent.click(screen.getByText(Config.navigationDefaults.navigationQuickLinks.navBarPlanText));
+
+    expect(window.open).toHaveBeenCalledWith(
+      Config.navigationDefaults.navigationQuickLinks.navBarPlanLink,
+      "_blank",
+      "noopener noreferrer"
+    );
+  });
+
+  it("renders the operate quick link and redirects correctly", async () => {
+    render(<NavBarDesktopQuickLinks />);
+    fireEvent.click(screen.getByText(Config.navigationDefaults.navigationQuickLinks.navBarOperateText));
+
+    expect(window.open).toHaveBeenCalledWith(
```

Comment:
pebble - because your not resetting the mocks this result can be a carry over from a previous test.